gnovm/cmd/gno/doc.go

```diff
@@ -74,8 +78,24 @@ func execDoc(cfg *docCfg, args []string, io *commands.IO) error {
 	if cfg.rootDir == "" {
 		cfg.rootDir = guessRootDir()
 	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("could not determine working directory: %w", err)
+	}
+
+	rd, err := gnomod.FindRootDir(wd)
+	if err != nil && !errors.Is(err, gnomod.ErrGnoModNotFound) {
+		return fmt.Errorf("error determining root gno.mod file: %w", err)
+	}
+	var modDirs []string
+	if rd != "" {
+		modDirs = append(modDirs, rd)
+	}
```

Comment:
redundant check `if rd != ""`? we already checked `if err != nil`. Means `rd` can never be `""`?
```suggestion
	modDirs := []string{rd}
```

Reply:
You're right